lib/system/iterators_1.nim

```diff
@@ -104,10 +104,12 @@ template dotdotImpl(t) {.dirty.} =
     ##
     ## See also:
     ## * [..<](#..<.i,T,T)
-    var res = a
-    while res <= b:
-      yield res
-      inc(res)
+    if a <= b:
```

Comment:
Great, so now every time I write ``..`` I get two checks in the asm because I might count up to high(int32). Not acceptable to me.